### PR TITLE
Solve Week 1 Training

### PR DESCRIPTION
**Pull request by SecuredEevee**
1주차 Pull Request Test

### CODE REVIEW

File: HWP/1_recursion/17478_WhatisRecursiveFunction.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+    print("_" * 4 * cnt + '"�� ����. �������� �� �� ���⿡ �̼��� ��� ���� ����� ������ �־��.')
+    print("_" * 4 * cnt + '��� ����� ��� �� ���ο��� ��� ��� �߰�, ��� ����Ӱ� ����� �־���.')
+    print("_" * 4 * cnt + '���� ��� ��κ� �ǾҴٰ� �ϳ�. �׷��� ��� ��, �� ���ο��� �� ���� ã�ƿͼ� �����."')
+
+    sol(num-1, cnt + 1 )
+    print('_' * 4 * cnt + '��� �亯�Ͽ���.')
+
+if __name__ == "__main__":
+    n = int(input())
```
Comment (SecuredEevee) [inline HWP/1_recursion/17478_WhatisRecursiveFunction.py:16]: Review Test

Comment (kiwon94) [reply HWP/1_recursion/17478_WhatisRecursiveFunction.py:16]: good

**Review by kiwon94 — APPROVED** [review]
good

File: HWP/1_recursion/17478_WhatisRecursiveFunction.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    print("_" * 4 * cnt + '"����Լ� ������?"')
+    if num == 0:
+        print("_" * 4 * cnt + '"����Լ�� �ڱ� �ڽ�� ȣ���ϴ� �Լ���"')
+        print('_' * 4 * cnt + '��� �亯�Ͽ���.')
+        return
+
+    print("_" * 4 * cnt + '"�� ����. �������� �� �� ���⿡ �̼��� ��� ���� ����� ������ �־��.')
+    print("_" * 4 * cnt + '��� ����� ��� �� ���ο��� ��� ��� �߰�, ��� ����Ӱ� ����� �־���.')
+    print("_" * 4 * cnt + '���� ��� ��κ� �ǾҴٰ� �ϳ�. �׷��� ��� ��, �� ���ο��� �� ���� ã�ƿͼ� �����."')
+
```
Comment (kiwon94) [inline HWP/1_recursion/17478_WhatisRecursiveFunction.py:11]: comment test

**Pull request merged**